scenarioo-client/app/diffViewer/screenshotTitle/screenshotTitle.html

```diff
@@ -5,9 +5,7 @@ <h3>
     <span ng-if="!alias">{{build.name}}</span>
 
     <!-- Date and Revision as icon tooltip -->
-    <i class="icon-info-sign"
-       uib-tooltip="{{build.date | scDateTime}} {{build.revision}}"
-       tooltip-placement="bottom">
-    </i>
-
+    <span uib-tooltip="{{build.date | scDateTime}} {{build.revision}}" tooltip-placement="bottom">
```

Comment:
I think it is good practice to break attributes on separate lines ---> helps to read and merge.

Reply:
@sascha-kaufmann just for the future, it is ok for now